src/com/mambu/apisdk/Protocol.java

```diff
@@ -0,0 +1,11 @@
+package com.mambu.apisdk;
+
+/**
+ * Enum that stores application protocols that can be used within the project.
+ *
+ */
+public enum Protocol {
+
+	HTTP, HTTPS
+
```

Comment:
Does it need to be in a separate java file? If yes, may be it can be moved into a "model" package from the top level?